ghost/i18n/locales/it/ghost.json

```diff
@@ -2,15 +2,15 @@
     "All the best!": "A presto!",
     "Complete signup for {{siteTitle}}!": "Completa l'iscrizione a {{siteTitle}}!",
     "Complete your sign up to {{siteTitle}}!": "Completa la tua iscrizione a {{siteTitle}}!",
-    "Confirm email address": "Conferma l’indirizzo email",
-    "Confirm signup": "Conferma l’iscrizione",
+    "Confirm email address": "Conferma l'indirizzo email",
+    "Confirm signup": "Conferma l'iscrizione",
```

Comment:
It looks like the ’ has been changed to '.  In most languages, ’ is preferred. Is that an accidental change, or deliberate?